Documentation/kubernetes-on-baremetal.md

```diff
@@ -1,13 +1,34 @@
 
-This guide walks a deployer through launching a multi-node Kubernetes cluster on bare metal servers running CoreOS.
-After completing this guide, a deployer will be able to interact with the Kubernetes API from their workstation using the kubectl CLI tool.
+This guide walks a deployer through launching a multi-node Kubernetes cluster on bare metal servers running CoreOS. After completing this guide, a deployer will be able to interact with the Kubernetes API from their workstation using the kubectl CLI tool.
 
-## Deployment Prerequisites
+## Deployment Requirements
 
-### CoreOS Installation
+### CoreOS Version
 
-For all nodes running Kubernetes components (masters & workers), you must use a CoreOS version 962.0.0+ for the `kubelet-wrapper` script to be present in the image. If you wish to use an earlier version (e.g. from the 'stable' channel) see [kubelet-wrapper](kubelet-wrapper.md) for more information.
+All Kubernetes controllers and nodes must use a CoreOS version 962.0.0+ for the `kubelet-wrapper` script to be present in the image. If you wish to use an earlier version (e.g. from the 'stable' channel) see [kubelet-wrapper](kubelet-wrapper.md) for more information.
+
+### Kubernetes Pod Network
+
+These guides use [flannel][coreos-flannel] as a software-defined overlay network to manage routing of the [pod network][pod-network]. However, bare metal is a common platform where a self-managed network may be used, due to the flexibility provided by physical networking gear. One common alternative is to use [Calico][calico-networking] networking, avoiding the use of an overlay and allowing interop with physical networking gear using BGP.
+
+See the [Kubernetes networking](kubernetes-networking.md) documentation for more information on self-managed networking options.
+
+[coreos-flannel]: https://coreos.com/flannel/docs/latest/flannel-config.html
+[calico-networking]: https://github.com/projectcalico/calico-containers
+[pod-network]: https://github.com/kubernetes/kubernetes/blob/release-1.2/docs/design/networking.md#pod-to-pod
+
+## Automated Provisioning
+
+Booting and provisioning CoreOS clusters can be automated using the [coreos-baremetal](https://github.com/coreos/coreos-baremetal) project. It includes:
+
+* Guides for configuring an network boot environment with iPXE/GRUB
+* An HTTP/gRPC service for booting and provisioning machines. Match machines by their hardware attributes and serve templated [Ignition](https://coreos.com/ignition/docs/latest/c) configs or cloud-configs.
+* [Example](https://github.com/coreos/coreos-baremetal/tree/master/examples) clusters including an [etcd cluster](https://github.com/coreos/coreos-baremetal/blob/master/Documentation/getting-started-rkt.md), [multi-node Kubernetes cluster](https://github.com/coreos/coreos-baremetal/blob/master/Documentation/kubernetes.md), and [self-hosted](https://github.com/coreos/coreos-baremetal/blob/master/Documentation/bootkube.md) Kubernetes cluster. 
+
```

Comment:
Much improved. Only issue I can spot, do we need to have a main call to action here? Like "Get started with automated provisioning" or similar. Maybe even unlink a few of the examples so we can funnel you into a single spot.